src/registrar/templates/emails/status_change_rejected.txt

```diff
@@ -29,4 +29,4 @@ requesting a .gov domain.
 The .gov team
 Contact us: <https://get.gov/contact/>
 Visit <https://get.gov>
-{% endautoescape %}
\ No newline at end of file
+{% endautoescape %}
```

Comment:
Git is really weird sometimes with this stuff. I first thought maybe a zero-width character was introduced but nope - I think its just git being git